library/career-post.html

```diff
@@ -127,4 +127,4 @@
 </div>
 
 
-${require('./layouts/footer.html')}
+${require('./layouts/footer-blog.html')}
```

Comment:
standart footer was footer-big-menu, but we need to have footer-blog(i changed it on footer-dark)